docs/tables/circleci_insights_workflow.md

```diff
@@ -0,0 +1,19 @@
+# Table: circleci_insights_workflow
+
+Get Insights for your workflows
+
+## Examples
+
+### Get average duration and deployment count of a project for each month
+
+```sql
+SELECT 
```

Comment:
@leonid-panich could you please update the example query to follow the pattern used in other table docs?

To format the query, we recommend using https://www.freeformatter.com/sql-formatter.html#before-output (2 spaces per indent level)